controllers/admin/AdminSaferPayOfficialPaymentController.php

```diff
@@ -235,4 +267,28 @@ protected function initForm()
             ],
         ];
     }
+
+    private function getPaymentMethods(): ?array
+    {
+        try {
+            /** @var \Invertus\SaferPay\Service\SaferPayObtainPaymentMethods $saferPayObtainPaymentMethods */
+            $saferPayObtainPaymentMethods = $this->module->getModuleContainer()->get(SaferPayObtainPaymentMethods::class);
+
+            return $saferPayObtainPaymentMethods->obtainPaymentMethodsNamesAsArray();
+        } catch (\Exception $exception) {
+            /** @var \Invertus\SaferPay\Service\SaferPayExceptionService $exceptionService */
+            $exceptionService = $this->module->getModuleContainer()
+                ->get(\Invertus\SaferPay\Service\SaferPayExceptionService::class);
+            $saferPayErrors = json_decode($this->context->cookie->saferPayErrors, true);
+            $saferPayErrors[] = $exceptionService->getErrorMessageForException(
+                $exception,
+                $exceptionService->getErrorMessages()
+            );
+            $this->context->cookie->saferPayErrors = json_encode($saferPayErrors);
+
+            $this->errors[] = $this->l('To see available payment methods, you must connect to your SaferPay account.');
+
+            return null;
```

Comment:
Maybe should return empty array and then there would be no need to check if it is null, foreach'ing empty array would display nothing